integration-tests/kafka-avro/src/main/resources/application.properties

```diff
@@ -7,4 +7,3 @@ quarkus.kafka.health.enabled=true
 
 # TODO: this should not be needed, but Avro does not seem to use the correct CL
 # This will also cause dev mode issues
```

Comment:
I'd remove the comment as well, it's outdated with this fix.

Reply:
I did it and force pushed.

Reply:
oops, I just force pushed over your force push

Reply:
Hey, they are MY working hours! You should be in bed.